### PR TITLE
Hello logo

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2016-07-05 at 14 19 38

Fixes https://github.com/CartoDB/cartodb/issues/8473.
CR: @alonsogarciapablo 
cc @saleiva @urbanphes @piensaenpixel

### CODE REVIEW

**Comment by xavijam** [comment]
@nobuti, using the overlays in order to render the logo, but the name has changed, let's get rid of the cartodb prefix. So just **logo**, will it fit with your map options behaviour?

**Comment by nobuti** [comment]
Yes! 🚀

File: src/api/create-vis.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ var createVis = function (el, vizjson, options) {
     apiKey: options.apiKey,
     authToken: vizjson.auth_tokens && vizjson.auth_tokens.length && vizjson.auth_tokens[0],
     showLegends: options.legends === true || vizjson.legends === true,
+    showLogo: options.logo === undefined || options.logo === true || vizjson.logo === true,
```
Comment (alonsogarciapablo) [inline src/api/create-vis.js]: We could also add a default value for showLogo [here](https://github.com/CartoDB/cartodb.js/blob/66740c833adfeb8729e0bfe6966f953a17e5e27c/src/api/create-vis.js#L7). and remove the `options.logo === undefined`...

File: src/vis/vis-view.js [hunk 2]
```diff
@@ -223,6 +223,14 @@ var Vis = View.extend({
       return overlay.order === null ? Number.MAX_VALUE : overlay.order;
     });
 
+    // if logo is not included, let add it :)
+    var overlayNames = _.pluck(overlays, 'type');
+    if (!_.contains(overlayNames, 'logo')) {
```
Comment (alonsogarciapablo) [inline src/vis/vis-view.js]: Take a look at [this part of the code where we're already adding new overlays to the vizjson object](https://github.com/CartoDB/cartodb.js/blob/v4/src/api/create-vis.js#L86). Perhaps worth adding this overlay there too? (just to have it in the same place)

**Comment by xavijam** [comment]
I'll take those comments into account in other PR, don't worry @nobuti :).

**Pull request merged**